Sprint-2/implement/contains.js

```diff
@@ -1,3 +1,14 @@
-function contains() {}
+function contains(mainObject, char) {
+  for (const key of Object.keys(mainObject)) {
+    if (key.includes(char)) {
+      return true;
+    } else return false;
+  }
+  if (Object.keys(mainObject).length == 0 || typeof mainObject != "object")
```

Comment:
- It is better to check and reject invalid values at the start of the function block. This way, we can assure the remaining code in the function does not have to deal with invalid values.

Reply:
ok, thank you CJ. I swapped it and put "return false" at the top.